lib/mayaUsd/ufe/private/UfeNotifGuard.h

```diff
@@ -40,20 +40,20 @@ class InPathChange
 	static bool inGuard;
 };
 
-//! \brief Helper class to scope when we are in an add or remove reference operation.
-class InAddOrRemoveReference
+//! \brief Helper class to scope when we are in an add or delete operation.
+class InAddOrDeleteOperation
```

Comment:
As we discussed, removed the special case for add/remove reference and instead made a special case for add/delete of prims.